catroid/src/main/java/org/catrobat/catroid/content/bricks/GoToBrick.java

```diff
@@ -152,19 +148,19 @@ public View getPrototypeView(Context context) {
 	public List<SequenceAction> addActionToSequence(Sprite sprite, SequenceAction sequence) {
 		sequence.addAction(sprite.getActionFactory().createGoToAction(sprite, destinationSprite, spinnerSelection));
 
-		return null;
+		return Collections.emptyList();
```

Comment:
Not it is not necessary, it's just suggested to not return or pass around null and use empty collections instead. And since the file was already open, I "left it a bit more nice than I found it" ;)